php-practice/section11.php

```diff
@@ -174,15 +187,20 @@ public function bark() {
 $myDog->genus = 'Canine';
 $myDog->species = 'Scary';
 $myDog->barkIsBiggerThanBite = false;
-$myDog->bark(); // What will this print out? ANSWER:
+$myDog->bark(); // What will this print out? ANSWER: ...You better run, man... This is because we have barkIsBiggerThanBite set to False so when function bark() runs the if function runs it sees 
+                /* that $this->barkIsBiggerThanBite is false and goes to the else portion and echo out the string.
 
 /**
  * PROBLEM 4
  *
  * Create a $kangaroo variable as a `new Mammal()` object. Try to set the
- * barkIsBiggerThanBite property on it -- what happens? Why?
+ * barkIsBiggerThanBite property on it -- what happens? Why? When I tried it it said " Call to undefined method Mammal::bark()"
```

Comment:
The instructions here don't say anything about calling the bark function.   This problem is actually all about setting the barkIsBiggerThanBite property.

What I want you to notice is that you can set that property even if it wasn't defined in the class. Notice that it didn't fail!

Basically what this means is, objects can have random, arbitrary properties created on them even if the properties are not defined in the class.